install.sh

```diff
@@ -85,55 +170,104 @@ if !([ -f /etc/ros/rosdep/sources.list.d/20-default.list ]); then
 fi
 rosdep update
 
+instlog "Installing Sub8 ROS dependencies"
+sudo apt-get install -qq binutils-dev
+sudo apt-get install -qq ros-kinetic-ompl
+sudo apt-get install -qq ros-kinetic-spacenav-node
 
-#=====================================#
-# Workspace and Sub Repository Set Up #
-#=====================================#
+# Get older version of Google protobuf for Gazebo
+if !( protoc --version | grep "libprotoc 2.6.1" ); then
+    instlog "Installing Google protobuf v2.6.1"
+    wget https://github.com/google/protobuf/releases/download/v2.6.1/protobuf-2.6.1.tar.gz
+    tar -xvzf protobuf-2.6.1.tar.gz
+    cd protobuf-2.6.1
+    ./configure
+    make
+    make check
+    sudo make install
+    cd ..
+    sudo rm -r protobuf-2.6.1
+    sudo rm -r protobuf-2.6.1.tar.gz
+else
+    instlog "Detected correct version of protoc"
+fi
 
 # Set up catkin workspace directory
 if !([ -f $CATKIN_DIR/src/CMakeLists.txt ]); then
     instlog "Generating catkin workspace at $CATKIN_DIR"
-    mkdir -p "$CATKIN_DIR/src"
-    cd "$CATKIN_DIR/src"
+    mkdir -p $CATKIN_DIR/src
+    cd $CATKIN_DIR/src
     catkin_init_workspace
-    catkin_make -C "$CATKIN_DIR"
 else
     instlog "Using existing catkin workspace at $CATKIN_DIR"
+    cd $CATKIN_DIR/src
 fi
 
-# If we're in the Semaphore-ci, we should run catkin_make in the actual build thread
-if (env | grep --quiet "SEMAPHORE=true"); then
-    mv ~/Sub8 "$CATKIN_DIR/src"
-fi
+CMAKE_PREFIX_PATH=$CMAKE_PREFIX_PATH:/opt/ros/kinetic
+git clone https://github.com/ros-drivers/driver_common
+catkin_make -C $CATKIN_DIR -B
```

Comment:
Yeah... the problem was that the Camera1394 depends on the driver_base... But during compilation Camera1394 would fail to build because it was looking for driver_base dependences which didn't exist yet. No clue how to fix this... My solution was to build driver_common first. Maybe build driver_common into the /ros/kinetic